docs/src/main/asciidoc/hibernate-reactive.adoc

```diff
@@ -354,6 +354,38 @@ public class SomeTest {
 ----
 ====
 
+[[hr-named-datasource]]
+==== Named data sources
+
+Hibernate Reactive supports having named data sources
+
+[source,properties]
+.Example `{config-file}`
+----
+# datasource configuration
+quarkus.hibernate-orm.datasource = named-datasource
+quarkus.datasource."named-datasource".reactive", true
+quarkus.datasource."named-datasource".db-kind", postgresql
+quarkus.datasource."named-datasource".username", quarkus_test
+quarkus.datasource."named-datasource".password", quarkus_test
+
+quarkus.datasource.reactive.url = vertx-reactive:postgresql://localhost/quarkus_test <1>
```

Comment:
You could perhaps follow the same best practices you implemented in https://github.com/quarkusio/quarkus-quickstarts/pull/1520 ? :)
That would mean using a `%prod.` prefix here.

Reply:
https://github.com/quarkusio/quarkus/pull/47631/commits/f939425a69d9ee95b45112d1c2e045afaaae5c10